### PR TITLE
Latest tx info

### PR DESCRIPTION
**Pull request by tdroxler**
* Reduce max page limit to 20 and default to 10
* Add /addresses/<address>/latest-transaction endpoint

### CODE REVIEW

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

File: app/src/main/scala/org/alephium/explorer/api/AddressesEndpoints.scala [hunk 1]
```diff
@@ -113,7 +113,7 @@ trait AddressesEndpoints extends BaseEndpoint with QueryParams {
   val listAddressTokens: BaseEndpoint[(Address, Pagination), ArraySeq[TokenId]] =
     addressesTokensEndpoint.get
       .out(jsonBody[ArraySeq[TokenId]])
-      .in(paginator(defaultLimit = 100))
+      .in(paginator(limit = 100))
```
Comment (polarker) [inline app/src/main/scala/org/alephium/explorer/api/AddressesEndpoints.scala:122]: Does it mean the default limit is 100 for tokens?

**Review by polarker — APPROVED** [review]
LGTM

File: app/src/main/scala/org/alephium/explorer/api/AddressesEndpoints.scala (hunk 1 above)
Comment (tdroxler) [reply app/src/main/scala/org/alephium/explorer/api/AddressesEndpoints.scala:122]: yes, this endpoint is just returning `TokenId` stored in one table, no heavy query like tx. But we could reduce it too if you prefer.

Comment (polarker) [reply app/src/main/scala/org/alephium/explorer/api/AddressesEndpoints.scala:122]: maybe run some benchmark on this to see the latency

Comment (tdroxler) [reply app/src/main/scala/org/alephium/explorer/api/AddressesEndpoints.scala:122]: Reading 100 or 10 row from DB is equivalent. 
image
image

Comment (polarker) [reply app/src/main/scala/org/alephium/explorer/api/AddressesEndpoints.scala:122]: Interesting. Then let's keep it unchanged for now

**Pull request merged**